app/controllers/claimToAdjustPoa/AmendablePOAController.scala

```diff
@@ -71,4 +76,16 @@ class AmendablePOAController @Inject()(val authorisedFunctions: AuthorisedFuncti
             showInternalServerError(isAgent)
         }
     }
+
+  private def isSubsequentAdjustmentAttempt(paymentOnAccount: PaymentOnAccountViewModel)(implicit user: MtdItUser[_]): Future[Boolean] =
```

Comment:
this method is a good candidate to be moved within ClaimToAdjustService as also this is something we need to tests